doc/update-bad-bootloader-v2.md

```diff
@@ -0,0 +1,113 @@
+# SMC update with bad bootloader v2 (July 13th)
```

Comment:
The name of this file may suggest that this document concerns updating the bootloader, while it in reality concerns how to actually help the bad bootloader to do its job properly.

The title is better.

Maybe rename the file to "update-smc-using-bad-bootloader-v2.md", as this document how to program main SMC firmware using this particular bootloader version. The title can eventually be adjusted once we find a good file name. Not sure if the date provides a value, as there is even a changelog at the bottom. Maybe add one new entry to the changelog, "Version 2", and specify that it is based on version 1, changed to Markdown, simplified and adjusted/updated a bit.